JobConfig/digitize/prolog.fcl

```diff
@@ -54,18 +50,40 @@ Digitize: {
       mean : 1
       halfWidth : 0
     }
+# Trigger MC matching instances for each helix algorithm
+    TTAprKSFMC : {
+      @table::CommonMC.TTSelectRecoMC
+      KalSeedCollections  : [ "TTAprKSF" ]
+    }
+    TTTprDeKSFMC : {
+      @table::CommonMC.TTSelectRecoMC
+      KalSeedCollections  : [ "TTKSFDe" ]
+    }
+    TTMprDeKSFMC : {
+      @table::CommonMC.TTSelectRecoMC
+      KalSeedCollections  : [ "TTKSFMprDe" ]
+    }
+    TTCprDeKSFMC : {
+      @table::CommonMC.TTSelectRecoMC
+      KalSeedCollections  : [ "TTCalSeedFitDe" ]
+    }
+    MakeSS : {
+      @table::CommonMC.MakeSS
+      VDStepPointMCs : "compressDetStepMCs:virtualdetector"
+      AbsorberStepPointMCs : "compressDetStepMCs:protonabsorber"
+      TargetStepPointMCs : "compressDetStepMCs:stoppingtarget"
+    }
   }
-
   filters : {
-    # filter for Triggerable stream.  These should be events that might have
-    # triggered.  The default setting is a particle with enough tracker digis
+# filter for Triggerable stream.  These should be events that might have
+# triggered.  The default setting is a particle with enough tracker digis
     # in the right momentum range to be reconstructable as a helix
     # This filter should be overriden as needed for other kinds of primaries (straight cosmics, calo signals, ...)
     Triggerable : {
       module_type : StrawDigiMCFilter
       MinNDigis : 15
```

Comment:
Just as a separate question, is requiring 15 digis a bit strict? Could we expect some amount of 14 digi events to have pileup hits that lead to our signal or background process of interest triggering the event, which we would keep in downstream analysis?

Reply:
Keep in mind too that's the cut on the # of digis, the number of hits passing the preselection and time clustering will be less. IIRC these cuts correspond to >99% of tracks passing TrkQual>0.4.